modules/gzip/index.html

```diff
@@ -0,0 +1,108 @@
+---
+title: GZIP-kb Module
+---
+<!DOCTYPE html>
+<html lang="en">
+{% include header.html %}
+<body role="document">
+
+{% include navbar.html nav=site.data.navbar %}
+<div class="container" role="main">
+<h1>HTML-hul Module</h1>
+<a class="name" name="introduction">
+<h2>1 Introduction</h2>
+</a>
+<p>
+  The GZIP-kb module recognizes and validates the Gzip (GNU zip) format.
+  [<a href="/references#gzip">GZip</a>].
+</p>
+<p>
+  The module is invoked by the:
+</p>
+<blockquote>
+  <pre>
+    jhove ... -m GZIP-kb ...
+  </pre>
+</blockquote>
+<p>
+  command line option.
+</p>
+<p>
+  The GZIP-kb module recognizes and validates <a href="/references#gzip">GZip version 4.3</a>. It also supports multiple member GZip files. This module uses the <a href="/references#jwat">JWAT</a> library for GZip parsing.
+</p>
+
+<p>
+This module doesn't have configurable parameters.
+</p>
+
+<a class="name" name="coverage">
+  <h2>2 Coverage</h2>
+</a>
+<p>
+  The GZIP-kb module recognizes and validates the following public profiles:
+</p>
+<ul>
+  <li>
+    <a href="/references#gzip">RFC 1952</a>
+  </li>
+</ul>
+
+<a class="name" name="well-formedness">
+  <h2>3 Well-Formedness</h2>
+</a>
+<p>
+  The GZip module doesn't check the well-formedness.
+</p>
+
+<a class="name" name="validity">
+<h2>4 Validity</h2></a>
+<p>
+ The following criteria must be met by a GZip file for JHOVE to consider it valid:
+</p>
+<ul>
+<li> The file is well-formed
```

Comment:
You just stated before that the GZip module doesn't check the well-formedness. Do you mean the file is compliant with the RFC as suggested by the above property ?

Reply:
Well this is something I have doubts about. But it is implied in the WARC and the same way it is implemented in GZip.

The compliancy is evaluatued in JWAT and it is used for validation and well-formedness.

So in the code it is stated:
* Well-formedness is empty and validation is not (this is WARC, but the same happens in GZip: https://github.com/openpreserve/jhove/blob/dd25d0ff559931a68604f35fcdae337ddd4aae19/jhove-ext-modules/src/main/java/edu/harvard/hul/ois/jhove/module/WarcModule.java#L73-L74
* Same status is used for well-formedness and validity: https://github.com/openpreserve/jhove/blob/dd25d0ff559931a68604f35fcdae337ddd4aae19/jhove-ext-modules/src/main/java/edu/harvard/hul/ois/jhove/module/WarcModule.java#L230-L231
* but the compliancy is checked in JWAT: https://github.com/netarchivesuite/jwat/blob/eb87798130394893c1aed809cbe2365a9a52737e/jwat-warc/src/main/java/org/jwat/warc/WarcRecord.java#L458. You'll find GZip: https://github.com/netarchivesuite/jwat/blob/eb87798130394893c1aed809cbe2365a9a52737e/jwat-gzip/src/main/java/org/jwat/gzip/GzipReader.java#L185

I don't know if this is clear and if you want me to change anything. But thanks for the feedback! @tledoux 

Sam

Reply:
IMHO, wellFormedness means the file respects the grammar of the format while validity is granted when the file follows the constraint of a standard or a profile.
In the case of Gzip there is no place for distinction between these two notions : so I suppose it should be stated that Well-Formedness (which is what is checked here) is equivalent to Validity for this module.

Reply:
OK Thanks Thomas. I'll change it  I'll write that well-formedness is checked and in Validity the Well-formdness and Validity are equivalent. @tledoux